.travis.yml

```diff
@@ -19,13 +19,7 @@ before_script:
   > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
-- curl -L https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/bin/utils/openapi-generator-cli.sh
-  > ./openapi-generator-cli
-- chmod +x ./openapi-generator-cli
-- curl -L https://raw.githubusercontent.com/RedHatInsights/insights-api-common-rails/master/.travis_scripts/openapi-validator.sh
-  > ./openapi-validator.sh
-- chmod +x ./openapi-validator.sh
-- ./openapi-validator.sh
+- curl -sSL https://raw.githubusercontent.com/gmcculloug/insights-api-common-rails/master/.travis_scripts/openapi-validator.sh | bash -s
```

Comment:
Should this be `RedHatInsights` instead of `gmcculloug`?

Reply:
🤦  Thanks for catching this.

Reply:
Fixed